### PR TITLE
return terraform with version in the response

### PR DESCRIPTION
**Pull request by baixinsui**
Related to https://github.com/eclipse-xpanse/xpanse/pull/2034

### CODE REVIEW

File: src/main/java/org/eclipse/xpanse/terraform/boot/models/response/TerraformResult.java [hunk 1]
```diff
@@ -36,4 +33,6 @@ public class TerraformResult {
     @Schema(description = "Data of all other files generated by the terraform execution."
             + "The map key contains the file name and value is the file contents as string.")
     private Map<String, String> importantFileContentMap;
+    @Schema(description = "Terraform with version which executed the command.")
```
Comment (swaroopar) [inline src/main/java/org/eclipse/xpanse/terraform/boot/models/response/TerraformResult.java:36]: terraformVersionUsed
The version of the Terraform binary used to execute scripts

File: src/main/java/org/eclipse/xpanse/terraform/boot/models/plan/TerraformPlan.java [hunk 2]
```diff
@@ -17,10 +17,10 @@
...
 
-    @Schema(description = "Version of terraform")
-    String terraformVersion;
-
     @NotNull
     @Schema(description = "Terraform plan as a JSON string")
     String plan;
+
+    @Schema(description = "Terraform with version")
+    String terraformWithVersion;
```
Comment (swaroopar) [inline src/main/java/org/eclipse/xpanse/terraform/boot/models/plan/TerraformPlan.java:25]: terraformVersionUsed
The version of the Terraform binary used to execute scripts

File: src/main/java/org/eclipse/xpanse/terraform/boot/models/validation/TerraformValidationResult.java [hunk 3]
```diff
@@ -16,7 +18,13 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class TerraformValidationResult {
 
-    private String terraformVersion;
+    @NotNull
+    @Schema(description = "Defines if the Terraform scripts is valid.")
     private boolean valid;
+
+    @Schema(description = "Terraform with version which executed the command.")
+    private String terraformWithVersion;
```
Comment (swaroopar) [inline src/main/java/org/eclipse/xpanse/terraform/boot/models/validation/TerraformValidationResult.java:26]: terraformVersionUsed
The version of the Terraform binary used to execute scripts

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Comment by Alice1319** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**